types/canvas-confetti/canvas-confetti-tests.ts

```diff
@@ -44,7 +44,7 @@ confetti({
     origin: {
         y: 0.6
     },
-    shapes: ['square', 'circle', 'square']
+    shapes: ['square', 'circle', 'square', 'star']
```

Comment:
this says "square" twice

Reply:
Yes, because then you have twice as many squares as circles and stars. 

You should probably review the [official documentation](https://github.com/catdad/canvas-confetti#options). Questions on old type definitions pull requests aren't going to be high visibility.

> shapes Array<String>: An array of shapes for the confetti. The possible values are square, circle, and star. The default is to use both squares and circles in an even mix. To use a single shape, you can provide just one shape in the array, such as ['star']. You can also change the mix by providing a value such as ['circle', 'circle', 'square'] to use two third circles and one third squares.